internal/policies/privilege/privilege_test.go

```diff
@@ -131,3 +143,30 @@ func TestApplyPolicy(t *testing.T) {
 		})
 	}
 }
+
+// Creates a temporary directory with the default rules file for Polkit.
+//
+// This is needed to avoid the huge absolute paths that come from using t.TempDir() and can
+// eventually go over the maximum limit for paths.
+func createPolkitReservedPath(t *testing.T) string {
```

Comment:
Shouldn’t those directories and files be shipped as part of the test fixtures in some directories?

Reply:
I think maybe thus we should fix the test to not call `t.Tempdir` but using something like you do `os.MkdirTemp()` with a shorter path and commenting that this is to prevent overflowing there? (In your comment, I didn’t realize at first the fix was that you are doing `os.MkdirTemp` instead of it).

That way, we still have all fixtures at the same place.

Reply:
Hmm... :thinking:
It can be done, sure, but I didn't want to add too much to the test itself and we only have a single test case that requires this directory to be different. Also, the fixtures are made to resemble the structure in `/etc` and these assets don't belong there. If we want the file in `testdata/`, I think we should also switch the structure to resemble `/` instead. But is it worth it?

Reply:
I think it does, having things coming "from 2 places" looks weird to me, no?

Reply:
It depends... This is more of a system state than a fixture (which, to me, implies it's something we can control in some way), so I think it's fine to have it set in a different way/place. But I can update the assets to match the `/` structure we need if you think it's beneficial.

Reply:
`/etc` is a state, but as we are talking about `/usr` files here, we are more on mimicking the OS FS. I think it’s beneficial to take the time to update the structure so that we ship (as in other tests in adsys) in a single place the current emulated machine FS.

Reply:
Ack. Will do!